docs/source/markdown/podman-login.1.md

```diff
@@ -111,6 +111,14 @@ $ echo $testpassword | podman login -u testuser --password-stdin docker.io
 Login Succeeded!
 ```
 
+```
+$ podman login quay.io --verbose
+Username: flouthoc
```

Comment:
```suggestion
Username: myusername
```

Reply:
We try to keep examples like this generic, otherwise LGTM

Reply:
@TomSweeneyRedHat Resolved.